cluster-autoscaler/core/scale_down.go

```diff
@@ -1146,7 +1146,7 @@ func (sd *ScaleDown) deleteNode(node *apiv1.Node, pods []*apiv1.Pod,
 	sd.context.Recorder.Eventf(node, apiv1.EventTypeNormal, "ScaleDown", "marked the node as toBeDeleted/unschedulable")
 
 	// attempt drain
-	evictionResults, err := drainNode(node, pods, sd.context.ClientSet, sd.context.Recorder, sd.context.MaxGracefulTerminationSec, MaxPodEvictionTime, EvictionRetryTime, PodEvictionHeadroom)
+	evictionResults, err := drainNode(node, pods, sd.context.ClientSet, sd.context.Recorder, sd.context.MaxGracefulTerminationSec, sd.context.MaxPodEvictionTime, EvictionRetryTime, PodEvictionHeadroom)
```

Comment:
Can the const be removed then?

Reply:
Nevermind, it's used for the default further down I guess.

Reply:
It could, but I wanted to avoid the merge conflicts later.